src/metabase/search/api.clj

```diff
@@ -127,6 +127,9 @@
   - `search_native_query`: set to true to search the content of native queries
   - `verified`: set to true to search for verified items only (requires Content Management or Official Collections premium feature)
   - `ids`: search for items with those ids, works iff single value passed to `models`
+  - `display_type`: search for cards/models with specific display types
+  - `has_temporal_dimensions`: set to true to search for cards with temporal dimensions only
+  - `required_non_temporal_dimension_ids`: search for cards containing all specified non-temporal dimension field IDs
```

Comment:
Since required_non_temporal_dimension_ids is only used on postgresql and silently ignored on other appdb types, we should document that here in the top-level API.